src/pages/WordFrequency/components/DiffView/DiffView.tsx

```diff
@@ -65,42 +67,45 @@ export const DiffView = (props: DiffViewProps) => {
     fetchData()
   }, [nodeRelativePath, script])
 
-  const columnDefinitions: KSCUI.C.KsTableColumnDefinition[] = [
-    {

```

Comment:
in this case it should be easy to just modify line 68 to add another lambda and keep the rest same.